roles/edpm_telemetry/templates/ceilometer-agent-compute.json.j2

```diff
@@ -20,6 +20,15 @@
         "perm": "0600",
         "optional": true
       },
+{% if edpm_telemetry_tls_certs_enabled %}
+      {
+        "source": "/var/lib/openstack/cacerts/tls-ca-bundle.pem",
+        "dest": "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem",
```

Comment:
can just mount directly, don't need to use kolla to copy it

Reply:
Yes, thanks for the suggestion. I mounted it directly in the latest commit.